k8s/secret.yaml

```diff
@@ -27,9 +27,10 @@ metadata:
 data:
   AWS_ACCESS_KEY_ID: Y2hhbmdlbWU=
   AWS_SECRET_ACCESS_KEY: Y2hhbmdlbWU=
+  AWS_SESSION_TOKEN: Y2hhbmdlbWU=
   aws-account: Y2hhbmdlbWU=
-  aws-region: dXMtZWFzdC0x
-  aws_assume_role: YXJuOmF3czppYW06Ojk5OTk5OTk5OTpyb2xlL3JvbGUtYXJuLTIzNDIzNDIzZXRj
```

Comment:
I'm inconsistent with the underscore vs hyphen, the assume role is consumed with underscore's now: https://github.com/upmc-enterprises/registry-creds/blob/master/main.go#L70

Reply:
We could take a PR to standardize on hyphens, but changing to hyphens now would break the app.

Reply:
Oh.  The replication controller kept crashing for me _until_ I converted the underscores to hyphens. Perhaps I have an older version of minikube (`v0.25.0`)

Reply:
Are you enabling via the add-on? Might be swapped on how we setup in minikube.

Reply:
The replication controller in minikube seems to expect `aws-assume-role`:

https://github.com/kubernetes/minikube/blob/4d060d3bdba200f531fa00f04904e7076cb69c26/deploy/addons/registry-creds/registry-creds-rc.yaml#L57

Reply:
Yes, after I load the secrets I do `minikube addons enable registry-creds`